src/test/java/MainTest.java

```diff
@@ -0,0 +1,36 @@
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MainTest {
+
+    @Test
+    void shouldReturnProperSumOfFibonacciNumbers() {
+        int expectedHowManyNumbersShouldBeAddTogether = 33;
+        FibonacciEven.generateOnlyEvenFibonacciNumbersSum(expectedHowManyNumbersShouldBeAddTogether);
+        assertThat(expectedHowManyNumbersShouldBeAddTogether);
+    }
+
+    @Test
+    void shouldMakeUnableToUseZero() {
+        int unexpectedZero = 0;
+        int userChoice = 5;
+        FibonacciEven.generateOnlyEvenFibonacciNumbersSum(userChoice);
+        assertThat(unexpectedZero).isNotEqualTo(userChoice);
+    }
+    @Test
+    void shouldMakeUnableToUseNegativeNumbers(){
+        int testNumberFromUser = 20;
+        FibonacciEven.generateOnlyEvenFibonacciNumbersSum(15);
+        assertThat(testNumberFromUser)
+                .isNotNegative();
+    }
+    @Test
+    void shouldPreventHigherNumbersThan33 (){
+        int higherThanExpected = 34;
+        FibonacciEven.generateOnlyEvenFibonacciNumbersSum(38);
+        assertThat(32)
```

Comment:
Nie rozumiem za bardzo co ten test ma sprawdzać - że 32 jest mniejsze od 34? :)

Reply:
Chodzi o to, aby wypełnić tą część zadania
„Wylicz sumę parzystych wartości ciągu Fibonacciego których wartość nie przekracza 4 milionów”

Reply:
Tak, ale Twoja asercja nie ma sensu - zobacz co testujesz - mówisz "Upewnij się żę liczba 32 jest mniejsza od 34" Tutaj nie sprawdzasz w ogóle tego co dostałeś jako rezultat.